templates/accounts/signup.html

```diff
@@ -1,12 +1,20 @@
 {% extends 'base.html' %}
+{% load static %}
+
```

Comment:
reference the teachers' pages to add breadcrumbs navigation for this page and the two student/teacher signup pages: https://github.com/solved-chinese/app/blob/caa9f7cecb67ab6d9284fc156d8853e312224563/templates/classroom/class_detail.html#L3-L8

This page should be: Home > Sign Up
Teacher page: Home > Sign Up > Teacher Sign Up
Student page: Home > Sign Up > Student Sign Up

Reply:
Also reference Feishu's design guidelines for more detail